frontend/src/main/web/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "zanata-frontend",
+  "version": "1.0.0",
+  "description": "Zanata front-end package",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "js": "webpack",
+    "build": "NODE_ENV=production webpack -p --config webpack.prod.config.js --bail --display-error-details",
+    "start": "node server.js"
+  },
+  "license": "LGPL",
+  "devDependencies": {
+    "babel-core": "^4.3.0",
+    "babel-loader": "^4.0.0",
+    "css-loader": "^0.16.0",
+    "cssnext-loader": "^1.0.1",
+    "jest-cli": "~0.4.18",
+    "jsx-loader": "^0.12.2",
+    "react-hot-loader": "^1.1.1",
+    "react-tools": "~0.13.0",
```

Comment:
I see several `~` dependency ranges used in this (jest-cli, react-tools, object-assign). Is there a reason for this? If not, change them all to `^`.